image_client/PIC_undo_batch.py

```diff
@@ -81,7 +84,32 @@ def sql_tree_purger(self, database, table, timestamp1, timestamp2):
 
         cursor.close()
 
-    def casbotany_csv_purger(self, database: str, table: str, MD5: str):
+    def batch_log_clear(self, table, MD5: str):
+        """batch_log_clear: after removing imported records associated with a specific batch from
+                            the specify db, then clears the MD5 record in picbactch db to prevent confusion.
+           args:
+                table: the table with the batch_MD5 col from which records will be cleared.
+                MD5: the string of the MD5 code"""
+        try:
+            cursor = self.batch_db_connection.get_cursor()
+        except Exception as e:
+            self.logger.error(f"Connection Error: {e}")
+
+        sql = f'''DELETE FROM picbatch.{table} WHERE batch_MD5 = "{MD5}" '''
+        self.logger.info(f'running query: {sql}')
+        self.logger.debug(sql)
+        try:
+            cursor.execute(sql)
+        except Exception as e:
+            self.logger.error(f"Exception thrown while processing sql: {sql}\n{e}\n", flush=True)
+            self.logger.error(traceback.format_exc())
+
+        self.batch_db_connection.commit()
+
+        cursor.close()
+
+
+    def picturae_csv_undo(self, database: str, table: str, MD5: str):
         """casbotany_csv_purger: runs sql commands to database, to purge sql records created between two timestamps
```

Comment:
do we need the database parameter? we specify the database we connect to in the connection - that should be the only place we have to, so if we switch things up it's just a config file. makes code more modular.